pydantic/decorators.py

```diff
@@ -464,3 +464,8 @@ def dec(f: Callable[..., Any]) -> _decorators.PydanticDecoratorMarker[Any]:
         return dec
     else:
         return dec(__f)
+
+
+class AfterValidator:
+    def __init__(self, __func: _decorators.OnlyValueValidator | _core_schema.FieldValidatorFunction) -> None:
+        self.func = _decorators.make_generic_v2_field_or_general_validator(__func, mode='after')
```

Comment:
I wasn't sure where to put this, here is convenient because we have the `OnlyValueValidator` and other types, but I'm open to put it other places